site/content/docs/developing/domain.en.md

```diff
@@ -21,9 +27,6 @@ Currently, you can only use aliases under the domain you specified when creating
 
 We'll make this feature more powerful in the future by allowing you to import certificates and use any aliases!
 
-!!!info
```

Comment:
Why do we get rid of this info block? I think the rule still applies right?

Reply:
The info blocks felt like implementation detail that might not be interesting for users, that's why I removed it. Let me know if you disagree!